tests/common/plugins/loganalyzer/__init__.py

```diff
@@ -57,7 +70,9 @@ def loganalyzer(duthosts, request):
     store_la_logs = request.config.getoption("--store_la_logs")
     analyzers = {}
     should_rotate_log = request.config.getoption("--loganalyzer_rotate_logs")
-    if should_rotate_log:
+
+    is_modular_chassis = duthosts[0].get_facts().get("modular_chassis")
```

Comment:
Let's add some comments to this logic indicating this performance tuning

Reply:
@yejianquan  added some comments